docs/source/user_guide/key_concepts.md

```diff
@@ -0,0 +1,202 @@
+(user_guide/key_concepts)=
+# Key Concepts
+
+`derotation` is a modular package for reconstructing rotating image stacks acquired via line scanning microscopes. This page covers the core ideas, flow of information, and how the main modules interact.
+See the [API documentation](api_index) for full reference.
+
+## How derotation works
+
+Without derotation, movies acquired with a line scanning microscope are geometrically distorted — each line is captured at a slightly different angle, making it hard to register or interpret the resulting images.
+
+If the angle of rotation is recorded, `derotation` can reconstruct each frame by assigning a rotation angle to each acquired line and rotating it back to its original position. This is what is called derotation-by-line. This process incrementally reconstructs each frame and can optionally include shear deformation correction.
+
+![Derotation process](../_static/derotation_by_line.gif)
+
+## How to use derotation
+
+There are two main ways to use `derotation`:
+
+### 1. **Low-level core function**
+Use `derotate_an_image_array_line_by_line` to derotate an image stack, given:
```

Comment:
Can this link to the API docs?

Reply:
Good point, it should. Have to investigate how.